src/libraries/Microsoft.PowerFx.Core/Functions/FunctionScopeInfo.cs

```diff
@@ -200,6 +219,18 @@ public void CheckLiteralPredicates(TexlNode[] args, IErrorContainer errors)
                     }
                 }
             }
+        }
+
+        public virtual bool GetScopeIdent(TexlNode[] nodes, out DName[] scopeIdents)

```

Comment:
>GetScopeIdent [](http://example.com/codeflow?start=28&length=13)

Can you comment what the intention is here? And what does the return value mean? (I notice we return false with a non-null out param, which is unusual)

Maybe with an example?


The signature takes arrays, but we always return 1 item. 
Why is the output an array? Is it a parallel array to the input? 
Or could output jut be an IEnumerable? if it's just a list.